docstore/docstore.go

```diff
@@ -534,6 +537,12 @@ func (l *ActionList) String() string {
 	return "[" + strings.Join(as, ", ") + "]"
 }
 
+// AtomicWrites causes all following writes in the list to execute atomically.
```

Comment:
"to execute as a single atomic operation"

(to avoid confusion with the fact that each write already happens atomically by itself)